addons/rose/addon/components/rose/form/actions/index.hbs

```diff
@@ -4,17 +4,28 @@
 }}
 
 <Hds::ButtonSet class='rose-form-actions' ...attributes>
-  <Hds::Button
-    @text={{@submitText}}
-    disabled={{@submitDisabled}}
-    type='submit'
-  />
-  {{#if this.showCancel}}
+  {{#if (has-block 'actions')}}
+    {{yield
+      (hash
+        submitDisabled=@submitDisabled
+        cancelDisabled=@cancelDisabled
+        cancel=@cancel
```

Comment:
Do we need to pass these variables down as a convenience? Aren't they already available from the form?

My take on this is if we're yielding something custom that falls out of the normal use case, they can use whatever they need from the form component directly rather than us trying to pass down all the things they might need.

Any thoughts?

Reply:
That's a good point, `submitDisabled` and `cancelDisabled` are essentially the same thing as `form.disabled`. `cancelDisabled` does have a bit more of logic but we could leave that up to each use case. I think `cancel` might be the only one that is needed here since it uses `handleCancel` ?

Reply:
I think you could still do `form.handleCancel` if you needed access to that?

Reply:
Yup, I also think we don't need to pass it down as they are all already available through the `form` and can be accessed through that and keep this component simple
    `{{yield to='actions'}}`

Reply:
I agree with @ZedLi and @DhariniJeeva . I like the idea of calling those as API `form.handleCancel`. That is more intuitive (IMHO).

Reply:
Thank you @ZedLi , @DhariniJeeva , and @calcaide! Updated! 😎